### PR TITLE
Edit auto-pause behaviour

### PR DESCRIPTION
**Pull request by mesellings**
## Description

Modifications needed to the cluster auto-pause functionality as follows:

- Trial/Internal clusters are sent to sleep if no activity for 2 days
- Trial/Internal clusters with tag dev are sent to sleep 8h after wake
- Cluster tagged dev that are sleeping for over 30days get deleted

Update the documentation in https://docs.camunda.io/docs/components/concepts/clusters/

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [X] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [X] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [X] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [X] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [X] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by HanselIdes** [comment]
This is accurate, even though I think we'll need to change the behavior soon

**Review by HanselIdes — DISMISSED** [review]

**Comment by mesellings** [comment]
@HanselIdes Are we still waiting for feedback from the commercial side for the final decision about this UX bug, or should we go ahead and publish these docs changes?

**Comment by HanselIdes** [comment]
@mesellings You shouldn't wait, and you should feel empowered to make the decision.

Whether it's a bug or not is outside your scope as a technical writer. You're doing your job of ensuring the documentation reflects reality, and that's why I approved the docs changes.

I would rather make 2 changes with no hesitation than make no changes and continue thinking about it. If the rapid adoption product leadership (and that's not me anymore) changes their mind about autopause behavior, I don't imagine it's too difficult to make a quick update.

**Comment by mesellings** [comment]
@camunda/tech-writers If you can, please review this today so we can get it merged 🤞

File: docs/components/concepts/clusters.md
Comment (christinaausley) [inline docs/components/concepts/clusters.md:71]: Maybe outside of the scope of this issue, but could be worth noting or linking to on https://docs.camunda.io/docs/next/components/console/manage-clusters/manage-cluster/.

File: docs/components/concepts/clusters.md
Comment (christinaausley) [inline docs/components/concepts/clusters.md:83]:
```suggestion
To prevent auto-pause, [upgrade your Free Trial plan](https://camunda.com/pricing/) to a Starter or Enterprise plan.
```

Comment (christinaausley) [reply docs/components/concepts/clusters.md:83]: Would need to be adjusted across all of the docs here 👍

File: docs/components/concepts/clusters.md
Comment (mesellings) [reply docs/components/concepts/clusters.md:71]: Thanks, I'll note that as maybe part of a new PR 👍

**Comment by mesellings** [comment]
Thanks @christinaausley, change made, please re-review 👍

**Review by christinaausley — APPROVED** [review]

**Pull request merged**
